vispy/scene/widgets/widget.py

```diff
@@ -224,21 +244,21 @@ def add_grid(self, *args, **kwargs):
         """
         Create a new Grid and add it as a child widget.
 
-        All arguments are given to add_widget().
+        All arguments are given to Grid().
```

Comment:
From what I saw, the signature ~15 lines up is `add_widget(self, widget)` unless it is subclassed with more options elsewhere...